logger.js

```diff
@@ -50,7 +52,7 @@ function pinoLogger (opts, stream) {
   function loggingMiddleware (req, res, next) {
     req.id = genReqId(req)
     req.log = res.log = logger.child({req: req})
-    res.startTime = Date.now()
+    res[KEY_START_TIME] = res[KEY_START_TIME] || Date.now()
```

Comment:
My bad, I meant: `res[KEY_START_TIME] = Date.now()`. It can't come for anywhere as the Symbol is local